skycoin/skywire
# Changelog

Did you run `make format && make check`? yes

Fixes #948 

 Changes:	
- Write a shell  scripts to extract details about the latest release and pass it to goreleaser automatically.

How to test this PR:

1. Make sure that `git` and [goreleaser](https://goreleaser.com/install) are installed.
2. Checkout to a commit you would like to create a release against.
3. Run `go mod vendor` and `go mod tidy`.
4. Add details reagrding the relase as the top most release in the CHANGELOG.md file (maintaining the format)
5. Make sure that `git status` is in clean state. Commit all vendor changes and source code changes.
6. Uncomment `draft: true` in `.goreleaser.yml` if this is a test release.
7. Create a `git` tag with desired release version and release name: `git tag -a 0.1.0 -m "First release"`,
   where `0.1.0` is release version and `First release` is release name.
8. Push the created tag to the repository: `git push origin 0.1.0`, where `0.1.0` is release version.
9. [Issue a personal GitHub access token.](https://github.com/settings/tokens)
10. Run `make GITHUB_TOKEN=your_token custom-github-release`
11. [Check the created GitHub release.](https://github.com/skycoin/skywire/releases/)
